torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -780,7 +780,7 @@ Tensor* TensorExprKernel::computeValue(const torch::jit::Value* v) {
     case aten::div: {
       return computeTwoOperand(
           "aten_div", v, [](const ExprHandle& lhs, const ExprHandle& rhs) {
-            return boolToInteger(lhs) / boolToInteger(rhs);
+            return promoteIntegerToFloat(lhs) / promoteIntegerToFloat(rhs);
```

Comment:
`torch.rand(4) / torch.tensor([True])`
> tensor([0.5537, 0.8641, 0.2935, 0.0947])
`torch.tensor([True]) / torch.tensor([True])`
> tensor([1.])
I think we need to update promoteIntegerToFloat to use `includeBool = True`